lib/constants/platforms.ts

```diff
@@ -31,6 +32,7 @@ export const GITLAB_API_USING_HOST_TYPES = [
   'gitlab-tags',
   'gitlab-packages',
   'gitlab-changelog',
+  'pypi',
```

Comment:
can this cause issues when we've a gitlab hostrule without match host to apply to any pypi url? 🤔

Reply:
Do we ever have that? I think it would be user error if so